### ISSUE TITLE
Wrong newline encoding sievelib.managesieve.Client.getscript ?

### ISSUE THREAD

**Issue by tacruc** [tonioo/sievelib#95]
Thanks for the nice library. Saved me some time in creating sieve scripts to handle my emails. 

I had the follwing issue
```python
c = Client(server)
c.connect(user, password, starttls=True)
script = c.getscript('Open-Xchange')
c.havespace('Open-Xchange',len(new_script))
c.putscript('Open-Xchange',new_script)
```
Was not working c.havespace returned true, but putscript failed. The Problem was solved by
```python
c = Client(server)
c.connect(user, password, starttls=True)
script = c.getscript('Open-Xchange')
script.replace("\n","\r\n")
c.havespace('Open-Xchange',len(new_script))
c.putscript('Open-Xchange',new_script)
```

Therefore I'm wondering if this should be fixed in the library or if there is a good reason for this behavior.

**Comment by tonioo** [comment]
@tacruc With your fix, putscript works?

**Comment by tacruc** [comment]
Yes, but I only tested it against the sieve server of mailbox.org and have no broader background of how or why it works. It was just a good guess.

**Comment by derula** [comment]
Note: the parser isn't really set up to work with CRLF linebreaks, so if this behavior is changed, the parser might break. I particularly noticed this with multi-line strings (parser only recognized them if they end on `.\n`, but with CRLF they would end on `.\r\n`), but there might be other issues.

**Comment by derula** [comment]
Addendum: since the Sieve spec is pretty clear about only allowing `\r\n` linebreaks, I think the best way to comply would be to never convert it to Unix linebreaks to begin with. But this might require some non-trivial changes to the parser.
